test/document_test.rb

```diff
@@ -22,6 +22,10 @@ def add_to_foo key, value
   end
 end
 
+class TestFoo < Dolly::Document
+  property :default_test_property, class_name: String, default: 'FOO'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.